engine/src/main/java/com/agiletec/aps/system/services/page/PageDAO.java

```diff
@@ -737,17 +714,7 @@ public void movePage(IPage currentPage, IPage newParent) {
 		try {
 			conn = this.getConnection();
 			conn.setAutoCommit(false);
-			// position
-			int pos = 1;
-			IPage[] sisters = newParent.getChildren();
-			if (null != sisters && sisters.length > 0) {
-				IPage last = sisters[sisters.length - 1];
-				if (null != last) {
-					pos = last.getPosition() + 1;
-				} else {
-					pos = sisters.length + 1;
-				}
-			}
+			int pos = this.getLastPosition(newParent.getCode(), conn) + 1;

```

Comment:
Same note as above on position calculation.

Reply:
see the previous reply